src/status_im2/contexts/wallet/edit_account/view.cljs

```diff
@@ -9,15 +11,53 @@
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
+(defn show-toast
+  [{:keys [type theme]}]
+  (let [messsage (condp = type
+                   :name  :t/edit-wallet-account-name-updated-message
+                   :color :t/edit-wallet-account-colour-updated-message
+                   :emoji :t/edit-wallet-account-emoji-updated-message)]
+    (rf/dispatch [:toasts/upsert
+                  {:id         :edit-account
+                   :icon       :i/correct
+                   :icon-color (colors/resolve-color :success theme)
+                   :text       (i18n/label messsage)}])))
+
 (defn- view-internal
-  []
-  (let [account-name    (reagent/atom "Account 1")
-        account-color   (reagent/atom :purple)
-        account-emoji   (reagent/atom "🍑")
-        account-address "0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045"
-        on-change-name  #(reset! account-name %)
-        on-change-color #(reset! account-color %)
-        on-change-emoji #(reset! account-emoji %)]
+  [{:keys [theme]}]
+  (let [{:keys [name customization-color emoji address]
+         :as   account}      (rf/sub [:wallet/current-viewing-account])
+        edited-data          (reagent/atom {:name                name
+                                            :customization-color customization-color
+                                            :emoji               emoji})
+        account-name         (reagent/cursor edited-data [:name])
+        account-color        (reagent/cursor edited-data [:customization-color])
+        account-emoji        (reagent/cursor edited-data [:emoji])
+        on-change-name       (fn [edited-name]
+                               ;;validation needs to be performed on the name (#17372)
```

Comment:
maybe we can remove the comment, I think we are doing a good enough job with the project tracking on the board etc anyway :)

Reply:
Sure 😄